Dockerfile.base

```diff
@@ -136,7 +136,7 @@ RUN apt update &&\
 	wget https://packages.sury.org/php/apt.gpg -O /etc/apt/trusted.gpg.d/php-sury.gpg &&\
 	echo "deb https://packages.sury.org/php/ stretch main" > /etc/apt/sources.list.d/php-sury.list &&\
 	apt update &&\
-	apt-get install -y php7.4-cli php7.4-xml php7.4-mbstring &&\
+	apt-get install -y php7.4-cli php7.4-xml php7.4-mbstring docker-compose &&\
```

Comment:
I think we need a more up-to-date version, so might try using `sudo	curl -L "https://github.com/docker/compose/releases/download/1.28.6/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose` next

Reply:
Isn't that dependent on the `docker` version used in this image, which is depends on the `docker` version used in the host? How does docker-in-docker even work in TeamCity?

If possible, I'd like to stick to debian packages. Maybe we can change the base image from `node:14.16.0` (based on Debian `stretch`) to `node:14.16.0-buster`, that should give us `docker-compose 1.21.0-3` if I'm not mistaken.

Reply:
Gotcha! It works fairly well in GitHub actions for both Gutenberg and current ETK. The problem I ran into was that `wp-env` would fail with an error about incompatible docker compose versions. Hopefully `-buster` works :)